package.json

```diff
@@ -5,8 +5,9 @@
   "main": "app.js",
   "scripts": {
     "start": "node app",
-    "build": "node app @apostrophecms/asset:build",
-    "dev": "nodemon --delay 1000ms -x \"nodemon app.js\""
+    "dev": "nodemon --delay 1000ms -x \"nodemon app.js\"",
+    "build": "NODE_ENV=production node app @apostrophecms/asset:build",
+    "production-start": "NODE_ENV=production node app"
```

Comment:
I think `npm run prod` is [more common](https://github.com/laravel/laravel/blob/8.x/package.json) and feels cleaner. If we're to start having a standard command for production I suggest that be it.